packages/recommend-vdom/src/FacetsView.tsx

```diff
@@ -20,7 +19,7 @@ export function createFacetsView({ createElement, Fragment }: Renderer) {
         <ol className={cx('auc-Recommend-list', props.classNames.list)}>
           {props.items.map((item) => (
             <li
-              key={item.facetValue}
+              key={`${item.facetValue}`}
```

Comment:
Any reasons for this explicit casting?

Reply:
Oups i should have reverted that after the trending-facets fix!